File: src/workspaces/monkey/Eat.rb
if $WORKSPACE.has("mango") == true and $WORKSPACE.has("monkey") == true
	fruit = $WORKSPACE.get("mango");
	monkey = $WORKSPACE.get("monkey");

	fruit.setEaten(true);
	monkey.setSleeping(false);
	monkey.setPlaying(false);
	monkey.setEating(true);

	$WORKSPACE.putOnWorkspace("eaten");

	$PLAN_CONTEXT.setIsFinished(true);
end


			

File: tags/blackboard_rev78/src/apps/example/CocoTheMonkey/monkey/Eat.js
if (WORKSPACE.has("example.CocoTheMonkey.mango") === true && WORKSPACE.has("example.CocoTheMonkey.monkey") === true)
{	
	var fruit = WORKSPACE.get("example.CocoTheMonkey.mango");
	var monkey = WORKSPACE.get("example.CocoTheMonkey.monkey");

	fruit.eaten = true;
	monkey.sleeping = false;
	monkey.playing = false;
	monkey.eating = true;

	WORKSPACE.put("example.CocoTheMonkey.eaten");

	PLAN_CONTEXT.isFinished = true;
}